lib/resque/scheduler.rb

```diff
@@ -189,12 +188,15 @@ def handle_delayed_items(at_time = nil)
               items.each do |item|
                 log "queuing #{item['class']} [delayed]"
```

Comment:
btw, unrelated to your PR, but `log` doesn't actually log for us in production (because our log level is not verbose enough). Not sure if changing this would be too spammy. Wdyt about adding a line of logging after calling `next_delayed_items` with some details about how many items of which class we are going to enqueue?